website/docs/cloud-docs/agents/agent-pools.mdx

```diff
@@ -116,6 +116,19 @@ To view agent statuses, go to your organization's settings and click **Agents**.
 - **Errored**: The agent encountered an unrecoverable error or has been in an Unknown state for long enough that HCP Terraform considers it errored. This status may indicate that something interrupted the agent process, the process crashed, a permanent network partition exists, or another similar problem. If the agent was in the process of running an operation (such as a plan or apply), the agent marks that operation as errored. If the current agent process recovers, it exits immediately.
 - **Exited**: The agent exited normally and successfully informed Terraform of it doing so.
 
+
+## Agent job types
+
+| Job type name | Description                   | Relevant documentation                                                                                          | Terraform CLI equivalent |
```

Comment:
1. Sorry to nit, but could you remove the "Relevant documentation" column and instead make the descriptions **full sentences** with in-line references in the table? For accessibility reasons it's better to have links in context so that those using screen readers know where the link will lead [here is another example of a table](https://github.com/hashicorp/tutorials/blob/main/STYLE_GUIDE.md#markdown-vs-html).

So, for example, in the first description: 
`The agent is [applying a run](/terraform/cli/commands/apply) in a workspace.`

Let me know if that makes sense or if you have any other questions!

2. Do you think we need the "Terraform CLI equivalent" column if we have links to the accompanying docs? I'm inclined to say no, but curious what you think.